honeybee_radiance_command/options/rpict.py

```diff
@@ -0,0 +1,548 @@
+from .optionbase import (
+    OptionCollection,
+    BoolOption,
+    NumericOption,
+    StringOption,
+    StringOptionJoined,
+    IntegerOption,
+    TupleOption,
+)
+
+
+class RpictOptions(OptionCollection):
+    """rpict command options."""
+
+    __slots__ = (
+        "_vt",
+        "_vp",
+        "_vd",
+        "_vu",
+        "_vh",
+        "_vv",
+        "_vo",
+        "_va",
+        "_vs",
+        "_vl",
+        "_x",
+        "_y",
+        "_pa",
+        "_pj",
+        "_pm",
+        "_pd",
+        "_ps",
+        "_pt",
+        "_t",
+        "_w",
+        "_i",
+        "_u",
+        "_bv",
+        "_dt",
+        "_dc",
+        "_dj",
+        "_ds",
+        "_dr",
+        "_dp",
+        "_dv",
+        "_ss",
+        "_st",
+        "_av",
+        "_aw",
+        "_ab",
+        "_aa",
+        "_ar",
+        "_ad",
+        "_as_",
+        "_me",
+        "_ma",
+        "_mg",
+        "_ms",
+        "_lr",
+        "_lw",
+        "_am",
+    )
+
+    def __init__(self):
+        """rpict command options."""
+        self._vt = StringOptionJoined("vt", "view type perspective - default: vtv")
+        self._vp = TupleOption(
+            "vp", "view point - default: 0.000000 0.000000 0.000000", 3, float
+        )
```

Comment:
TupleOption takes a value before length. Here the class: https://github.com/ladybug-tools/honeybee-radiance-command/blob/c9b1c59fb1c17dcc84f3e1a9fc144acc8309fa43/honeybee_radiance_command/options/optionbase.py#L341

A quick check of the rtrace example should have helped you figure this out yourself: https://github.com/ladybug-tools/honeybee-radiance-command/blob/c9b1c59fb1c17dcc84f3e1a9fc144acc8309fa43/honeybee_radiance_command/options/rtrace.py#L77-L79

Reply:
So why are we not providing these values in NumericOption?

Reply:
Good question! Because we if we provide default values, which is what we used to do they will be printed out when you use this options. By not assigning them we allow Radiance to use its default values under the hood.

If we would have set the values here to (0, 0, 0) when one translates the Option to Radiance format they will get `-ma 0.0 0.0 0.0`. This approach can make very long Radiance options and it is hard to see which one are Radiance default and which ones are set by user. With the current approach you will only see the values that are set by user. All the options classes should return an empty string by default.

Intended behavior:

```python
options = RpictOptions()
print(options)   # should print an empty string

options.ab = 2
print(options)   # should print -ab 2
```

Reply:
Thanks! I like that.